Library/Formula/macvim.rb

```diff
@@ -67,6 +67,15 @@ def install
     elsif build.with? "python"
       ENV.prepend "LDFLAGS", `python-config --ldflags`.chomp
       ENV.prepend "CFLAGS", `python-config --cflags`.chomp
+      framework_script = <<-EOS.undent
+        import distutils.sysconfig
+        print distutils.sysconfig.get_config_var("PYTHONFRAMEWORKPREFIX")
```

Comment:
Which pythons is this intended to work with?
I think the full set would be {brew,osx,python.org}{2.6,2.7,3,3,3.4,HEAD}.

I'm not sure which of those support this sysconfig-var.